sormas-backend/src/main/java/de/symeda/sormas/backend/systemevent/SystemEventFacadeEjb.java

```diff
@@ -56,21 +58,20 @@ public SystemEventDto getLatestSuccessByType(SystemEventType type) {
 		CriteriaQuery<SystemEvent> cq = cb.createQuery(SystemEvent.class);
 		Root<SystemEvent> systemEventRoot = cq.from(SystemEvent.class);
 
-		cq.where(cb.equal(systemEventRoot.get(SystemEvent.STATUS), SystemEventStatus.SUCCESS));
+		final Predicate equalSystemEventStatus = cb.equal(systemEventRoot.get(SystemEvent.STATUS), SystemEventStatus.SUCCESS);
+		final Predicate equalSystemEventType = cb.equal(systemEventRoot.get(SystemEvent.TYPE), type);
```

Comment:
I think [this method](https://github.com/hzi-braunschweig/SORMAS-Project/blob/aea878b619c4c0e96938097bebfc230941e83e9e/sormas-backend/src/main/java/de/symeda/sormas/backend/systemevent/SystemEventFacadeEjb.java#L101) also needs a distinction by type.

Reply:
Did I really miss that? :see_no_evil:

Reply:
No, I did.